Content.Shared/Inventory/InventorySystem.Equip.cs

```diff
@@ -424,15 +424,22 @@ public bool CanAccess(EntityUid actor, EntityUid target, EntityUid itemUid)
             return false;
         }
 
+        bool droppedItems = false;
         foreach (var slotDef in inventory.Slots)
         {
             if (slotDef != slotDefinition && slotDef.DependsOn == slotDefinition.Name)
             {
                 //this recursive call might be risky
-                TryUnequip(actor, target, slotDef.Name, true, true, predicted, inventory, reparent: reparent);
+                if (TryUnequip(actor, target, slotDef.Name, true, true, predicted, inventory, reparent: reparent))
+                    droppedItems = true;
             }
         }
 
+        // we check if any items were dropped, and make a popup if they were.
+        // this doesn't guarantee that only 1 popup will be shown if multiple items were dropped in recursion.
+        if (droppedItems)
+            _popup.PopupPredicted(Loc.GetString("inventory-component-dropped-from-unequip"), target, target);
```

Comment:
```suggestion
            _popup.PopupClient(Loc.GetString("inventory-component-dropped-from-unequip"), target, target);
```
PopupPredicted will show to everyone, PopupClient will show only to the specific player.